pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1724,7 +1726,10 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 						Issue in Libvirt: https://gitlab.com/libvirt/libvirt/-/issues/608
 						once the issue is resolved we can remove mpx disablement
 		*/
-		if _, exists := existingFeatures["mpx"]; !exists && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostModel && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostPassthrough {
+
+		_, exists := existingFeatures["mpx"]
+		// skip the mpx CPU feature validation for s390x as it is not supported.
+		if !checks.IsS390X(runtime.GOARCH) && !exists && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostModel && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostPassthrough {
```

Comment:
Please replace `runtime.GOARCH` with the `Architecture` of `ConverterContext`.

Reply:
There is already an `isS390X` function in `converter.go`. Please use it in a manner consistent with the rest of the converter package.

Reply:
I have done the changes as you suggested.
@xpivarc @jschintag  can you please review it again.
Thanks